app/utils.py

```diff
@@ -193,6 +193,10 @@ def log_auth_activity(user, message, admin_only=True):
 
 
 def log_user(user, message):
+    # Guards against the reserved keyword under the `info` method.
+    if user.get("name") is not None:
+        user["user_name"] = user.get("name")
+        user.pop("name", None)
```

Comment:
this could shortened to:
`if "name" in user:
    user["user_name"] = user.pop("name")`